mcs/class/System.Drawing/System.Drawing/KnownColors.cs

```diff
@@ -284,6 +285,9 @@ public static Color FromKnownColor (KnownColor kc)
 			}
 			c.knownColor = n;
 			return c;
+#else
+			return Color.FromKnownColor (kc);
```

Comment:
We could instead move our implementation to `Color` as well so we don't need this ifdef at all.